shared/chat/conversation/input.desktop.js

```diff
@@ -52,12 +54,11 @@ class Conversation extends Component<void, Props, State> {
       return
     }
 
-    // Allow copy
-    if (ev.metaKey && ['Meta', 'c'].includes(ev.key)) {
-      return
+    const isPasteKey = ev.key === 'v' && (ev.ctrlKey || ev.metaKey)
```

Comment:
Is there a paste event we could handle directly? It might be that people with uncommon accessibility settings don't use the `v` key. (Though I guess it's hard to know what will actually work until someone files a bug and we can test it?)